src/main/java/uk/gov/justice/hmpps/casenotes/alertnotes/AlertCaseNoteReconciliationController.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.hmpps.casenotes.alertnotes
+
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import java.time.LocalDate
+import java.time.LocalDate.now
+
+@RestController
+@RequestMapping
+class AlertCaseNoteReconciliationController(private val reconciliationEventGenerator: ReconciliationEventGenerator) {
+  @Operation(hidden = true)
+  @ResponseStatus(HttpStatus.NO_CONTENT)
+  @PostMapping("/case-notes/alerts/reconciliation")
+  fun caseNoteReconciliation(
+    @RequestParam(required = false) from: LocalDate?,
+    @RequestParam(required = false) to: LocalDate?,
+  ) {
+    reconciliationEventGenerator.generateEventsFor(from ?: now().minusDays(1), to ?: now())
```

Comment:
Is it worth using yesterday 00:00 to today 00:00 i.e. a full 24 hour day rather than now? Could be a second or two missed here or there I think? Not a big deal though